aws/lambdas/LogsToElasticSearch_info/index.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable */
+
+/*
+/* This Lambda is taken from the magically-generated lambda that is created 
+/* in AWS Console when setting up a log subscription to our Elasticsearch Kibana domain.
+/* We need to host this lambda in order to manage it via Terraform.
+*/
+
+// v1.1.2
+var https = require('https');
+var zlib = require('zlib');
+var crypto = require('crypto');
+
+const ENV = process.env;
```

Comment:
This is passed in via the lambda_function Terraform config - it's the only thing that I changed about the lambda function that's auto-generated in AWS Console.